src/current/v23.2/cluster-virtualization-overview.md

```diff
@@ -11,15 +11,15 @@ docs_area: deploy
 
 {% include_cached new-in.html version="v23.2" %} This page gives an overview of _cluster virtualization_ in CockroachDB {{page.version.version}}. Cluster virtualization allows you to separate a cluster's _control plane_ from its _data plane_. The cluster's control plane manages cluster nodes and node-to-node traffic, while its data plane reads data from and writes data to the cluster's storage.
 
-Cluster virtualization is required for [Physical Cluster Replication]({% link {{ page.version.version }}/physical-cluster-replication-overview.md %}). Creating virtual clusters without the intent of using them as either a physical cluster replication source or target is not yet supported.
+Cluster virtualization is required for [**Physical cluster replication (PCR)**]({% link {{ page.version.version }}/physical-cluster-replication-overview.md %}). Creating virtual clusters without the intent of using them as either a PCR primary or standby is not yet supported.
```

Comment:
Technically it is supported but just in preview?

Reply:
True, it is supported in the the product, but do we intend to support users who create multiple virtual clusters? I think last release we wanted to avoid users creating multiple virtual clusters other than for PCR. Happy to remove the sentence though, if we don't think it is helpful.

Reply:
I have updated the language around this.